### PR TITLE
feat: added bernoulli_

### PR DESCRIPTION
**Pull request by jaskiratsingh2000**
#22976

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by jaskiratsingh2000** [comment]
@Mr-Niraj-Kulkarni Excitedly looking forward to get it reviewed and merged!

**Comment by Mr-Niraj-Kulkarni** [comment]
@jaskiratsingh2000. Can you please add the tests for it as well.

**Comment by jaskiratsingh2000** [comment]
@Mr-Niraj-Kulkarni will it be possible that I could submit a test from another PR?

**Comment by Mr-Niraj-Kulkarni** [comment]
> @Mr-Niraj-Kulkarni will it be possible that I could submit a test from another PR?

No. You must write the tests and the test should pass for the implemented function. That is complete procedure for implementing a function. Please take help of the tests implemented in the file  to implement the test. Also refer the ivy documentation for test implementation. Once done please let me know I will review it then. Thanks

**Comment by jaskiratsingh2000** [comment]
@Mr-Niraj-Kulkarni I have implemented the test for the specific function. Can you please review it now?

**Comment by tbeetech** [comment]
I will take it from here

**Comment by jaskiratsingh2000** [comment]
@Mr-Niraj-Kulkarni After tremendous debugging I figured out that the test case for which it is failing is not from the code I have implemented it seems to be a whole from other. So I feel this is good to go

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 1]
```diff
@@ -4235,6 +4235,40 @@ def test_torch_tensor_bernoulli(
...
 
+# bernoulli_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bernoulli_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+    ),
+    test_with_out=st.just(True),
```
Comment (umairjavaid) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:4246]: Remove this flag

Comment (umairjavaid) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:4246]: Let me know if that works

Comment (jaskiratsingh2000) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:4246]: @umairjavaid you mean removing "test_with_out=st.just(True)" line right?

**Comment by hmahmood24** [comment]
Closing this PR since it has been stale for more than 2 weeks now.

**Pull request closed without merging**